HousingManagementSystemApi.Tests/HelpersTests/CosmosAddressQueryHelperTests.cs

```diff
@@ -0,0 +1,64 @@
+namespace HousingManagementSystemApi.Tests.HelperTests
+{
+    using HACT.Dtos;
+    using Helpers;
+    using Microsoft.Azure.Cosmos;
+    using Moq;
+    using Xunit;
+
+    public class CosmosAddressQueryHelperTests
+    {
+        private CosmosAddressQueryHelper systemUnderTest;
+        private Mock<Container> containerMock;
+        private Mock<FeedIterator<PropertyAddress>> feedIteratorMock;
+
+        public CosmosAddressQueryHelperTests()
+        {
+            this.containerMock = new Mock<Container>();
+            this.systemUnderTest = new CosmosAddressQueryHelper(this.containerMock.Object);
+            this.feedIteratorMock = new Mock<FeedIterator<PropertyAddress>>();
+        }
+
+        [Fact]
+        public void Test_Query_Definition_returned_with_querytext()
+        {
+            // Arrange
+            this.feedIteratorMock.Setup(_ => _.HasMoreResults).Returns(false);
+            this.containerMock
+                .Setup(_ => _.GetItemQueryIterator<PropertyAddress>("TEST", null, null)) //It.IsAny<string>()
```

Comment:
Remove commented code
```suggestion
                .Setup(_ => _.GetItemQueryIterator<PropertyAddress>("TEST", null, null))
```